omegaconf/omegaconf.py

```diff
@@ -467,6 +467,23 @@ def clear_resolvers() -> None:
         BaseContainer._resolvers = {}
         register_default_resolvers()
 
+    @classmethod
+    def clear_resolver(cls, name: str) -> bool:
+        """Clear(remove) any resolver only if it exists. Returns a bool: True if resolver is removed and False if not removed.
+
+        .. warning:
+            This method can remove deafult resolvers as well.
+
+        :param name: Name of the resolver.
+        :return: A bool (``True`` if resolver is removed, ``False`` if not found before removing).
+        """
+        if cls.has_resolver(name):
+            _ = BaseContainer._resolvers.pop(name)
+            return not cls.has_resolver(name)
```

Comment:
This should never be False. I think we can trust dict pop to work.
```suggestion
            return True
```

Reply:
Ok. I see your point. Incorporating your suggestion. Thank you.

> As a side note, the reason I kept that line `return not cls.has_resolver(name)` was to avoid any edge case, where `dict.pop()` fails somehow. However, I must also acknowledge that I am not aware of any such edge cases yet. So, I presume, a simple `return True` should suffice.